doc/dev/index.rst

```diff
@@ -227,16 +243,63 @@ Pending Backport Fix has been merged, backport(s) pending
 Resolved         Fix and backports (if any) have been merged
 ================ ===========================================
 
-If you do not have permission to change the status yourself, don't worry:
-someone will probably change it for you, even without asking. You can also
-ask for assistance on IRC.
+Submitting patches
+==================
+
+There are two ways to submit patches to Ceph: 
+
+1. `GitHub`_ pull requests
+2. Patch submission via :code:`ceph-devel@vger.kernel.org`
+
+Both ways are described in `the file SubmittingPatches in the top-level
+directory of the source-code tree`_.
+
+.. _`the file SubmittingPatches in the top-level directory of the source-code tree`: 
+  https://github.com/ceph/ceph/blob/master/SubmittingPatches
```

Comment:
A reference to https://github.com/ceph/ceph/blob/master/CONTRIBUTING.rst instead ?